widgets/MapWidget/README

```diff
@@ -0,0 +1,16 @@
+MapWidget README Last Change: 07/13/2015 by greenertux (Tristan Hoffmann)
+
+### Intro ###
+This widget can be used to visualize GPS coordinates. For any list of documents of a certain entity that contain coordinates, you can get a shiny map.
+See sample.jpg for a preview.
+
+### How to use ###
+1) Create a custom widget in an entity list view in AppBuilder and paste the *.erb code into it
+2) Download jquery-jvectormap-1.2.2.min.js and jquery-jvectormap-world-mill-en.js from http://jvectormap.com/download/ and place them in your webserver directory (usually /path/to-your-installation/AppBuilder/wlp/usr/servers/AppBuilder/apps/AppBuilder/assets)
+3) Replace your-entity with the name of the entity you want to visualize
+4) Replace LocationLat and LocationLon with the field names containing latitude and longitude information
+
+### Feedback ###
+Feel free to use Github to report issues or suggestions.
+
+IBM internally you may be interested in this forum post as well: https://w3-connections.ibm.com/forums/html/topic?id=670f0c9a-37b4-44ff-be93-2fc6ccedd295&ps=25
```

Comment:
I would not include links to IBM internal websites in the public GitHub readme files.  The goal with the public sample code is to have things that look good and "just work" for customers.  A broken link with important information can be confusing and doesn't look good.

It would be better if you cleaned up whatever information is relevant from that thread and included it in your readme file here.